### ISSUE TITLE
`nodent-compiler` dependencies `acorn` just had a break change.

### ISSUE THREAD

**Issue by zoolsher** [MatAtBread/nodent#112]
In nodent-compiler package.json. `acron >= 2.5.2`. 
```json
  "name": "nodent-compiler",
  "version": "3.1.3",
  "dependencies": {
    "acorn": ">=2.5.2",
    "acorn-es7-plugin": ">=1.1.6",
    "source-map": "^0.5.6"
  },
```
Acron 5
![image](https://user-images.githubusercontent.com/9161085/45537852-0668a200-b838-11e8-858b-5a8e96fcc6f5.png)

Acron 6.0.1
![image](https://user-images.githubusercontent.com/9161085/45537762-c0134300-b837-11e8-994c-55238eeffd2e.png)

There is no more `acron/lib/walk.js`. But it was required in `nodent-compiler/lib/parser.js`

**Comment by matAtWork** [comment]
Yeah, that's a breaking change. I'll see what I can do to patch it.

Why did you close the issue?

**Comment by matAtWork** [comment]
...I've just published patches that limit acorn to v 2.5.2-5.7.3, which at least unbreaks stuff. Acorn 6 is different enough that it'd be some real work to fix
